compiler/testSuite/Fintop00.bal

```diff
@@ -0,0 +1,14 @@
+import ballerina/io;
+
+public function main() {
+  printBoolean(7 == 8); // @output 0
```

Comment:
This is not legal according to the Ballerina spec. https://ballerina.io/spec/lang/master/#section_6.29

> it is a compile time error if the intersection of the static types of the operands is empty

which in this case it is